python/python/lance/ray/sink.py

```diff
@@ -213,24 +219,43 @@ class LanceFragmentWriter:
     in distributed fashion.
 
     Parameters
+    ----------
+    uri : str
+        The base URI of the dataset.
+    transform : Callable[[pa.Table], Union[pa.Table, Generator]], optional
+        A callable to transform the input batch. Default is identity.
+    schema : pyarrow.Schema, optional
+        The schema of the dataset.
+    max_rows_per_file : int, optional
+        The maximum number of rows per file. Default is 1024 * 1024.
+    max_bytes_per_file : int, optional
+        The maximum number of bytes per file. Default is None.
```

Comment:
Default is 90GB, not None, right?
```suggestion
        The maximum number of bytes per file. Default is 90GB.
```

Reply:
Fixed